Client.Core/Flux/Internal/FluxCsvParser.cs

```diff
@@ -1,7 +1,11 @@
 using System;
+using System.Collections;
 using System.Collections.Generic;
 using System.Globalization;
 using System.IO;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
```

Comment:
Using `System.Threading.Tasks` is not required by the code and can be safely removed

```suggestion
```